controlplane/migrations/0068_late_wolf_cub.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE "organization_limits" ADD COLUMN "users" integer DEFAULT 1 NOT NULL;--> statement-breakpoint
+ALTER TABLE "organization_limits" ADD COLUMN "graphs" integer DEFAULT 1 NOT NULL;--> statement-breakpoint
+ALTER TABLE "organizations" ADD COLUMN "plan" text;--> statement-breakpoint
+ALTER TABLE "organizations" ADD COLUMN "stripe_customer_id" text;--> statement-breakpoint
```

Comment:
Please create a separate table for subscriptions and payment information. The `organizations` table should only contain basic information about the org. It already contains a few limits we will move soon to `organization_limits`